m1well.Expenses/CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Expenses Plugin Changelog
```

Comment:
The first line in `CHANGELOG` should be in the form

# m1well.Expenses Plugin Changelog